patch/llpcPatchInOutImportExport.cpp

```diff
@@ -2795,7 +2772,9 @@ Value* PatchInOutImportExport::PatchFsBuiltInInputImport(
     auto& builtInUsage = m_pContext->GetShaderResourceUsage(ShaderStageFragment)->builtInUsage.fs;
     auto& inOutUsage = m_pContext->GetShaderResourceUsage(ShaderStageFragment)->inOutUsage;
 
-    std::vector<Value*> args;
+    Attribute::AttrKind attribs[] = {
+        Attribute::ReadNone
```

Comment:
The same. Use { Attribute::ReadNone } in EmitCall

Reply:
Are you sure? `attribs` is used more than once, so if you ever need to change the list of attributes, you will have to make the same change in more than one place.